src/coreComponents/mesh/ElementSubRegionBase.hpp

```diff
@@ -19,19 +19,49 @@
 #ifndef GEOSX_MESH_ELEMENTSUBREGIONBASE_HPP_
 #define GEOSX_MESH_ELEMENTSUBREGIONBASE_HPP_
 
+#include "codingUtilities/EnumStrings.hpp"
 #include "mesh/ObjectManagerBase.hpp"
+
 namespace geosx
 {
 
 class NodeManager;
 class FaceManager;
 class MeshLevel;
-class DomainPartition;
+
 namespace constitutive
 {
 class ConstitutiveBase;
 }
 
+/**
+ * @brief Denotes type of cell/element shape
+ */
+enum class ElementType : integer
+{
+  Line,          ///< Two-node line segment
+  Triangle,      ///< Three-node triangle
+  Quadrilateral, ///< Four-node quadrilateral
+  Polygon,       ///< Polygonal element (2D surface in 3D space)
+  Tetrahedon,    ///< Four-node tetrahedral element
+  Pyramid,       ///< Five-node pyramid element
+  Prism,         ///< Six-node wedge element
+  Hexahedron,    ///< Eight-node hexahedral element
+  Polyhedron     ///< Polyhedral element
+};
+
+/// Strings for ElementType
+ENUM_STRINGS( ElementType,
+              "BEAM",
+              "C2D3",
+              "C2D4",
+              "Polygon",
+              "C3D4",
+              "C3D5",
+              "C3D6",
+              "C3D8",
+              "Polyhedron" );
+
```

Comment:
Do you think this would be better in its own header? I don't know that this information would ever be useful outside the context of the subregion?

Reply:
I moved it into its own header. It can probably reduce the include graph in a few places... for example in `FiniteElementDiscretization.hpp`